builders/guest/build_linux.sh

```diff
@@ -4,4 +4,18 @@ set -o errexit -o nounset -o pipefail
 # See https://github.com/CosmWasm/wasmvm/issues/222#issuecomment-880616953 for two approaches to
 # enable stripping through cargo (if that is desired).
 
-cargo build --release
+echo "Starting x86_64-unknown-linux-gnu build"
+export CC=clang
+export CXX=clang++
+cargo build --release --target x86_64-unknown-linux-gnu
+cp -rf target/x86_64-unknown-linux-gnu/release/libwasmvm.so artifacts
```

Comment:
we don't need the `-r`, right? What about the `-f`?

Reply:
That's correct, and I will remove both of them.